Mlem/API/Models/Users/APIPerson.swift

```diff
@@ -29,3 +29,9 @@ struct APIPerson: Decodable {
     let banExpires: Date?
     let instanceId: Int
 }
+
+extension APIPerson: Equatable {
+    static func == (lhs: APIPerson, rhs: APIPerson) -> Bool {
+        lhs.actorId == rhs.actorId
```

Comment:
This looks ok, but why was it necessary? .id works fine too, since every person has an unique IDs in the server we're talking with